test/unittests/capi_test.cpp

```diff
@@ -0,0 +1,14 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <fizzy/fizzy.h>
+#include <gtest/gtest.h>
+
+TEST(capi, fizzy_validate)
```

Comment:
#533 does not use the `fizzy_` prefix here. Do we want to follow?

Reply:
I'm forward-following now.